resources/js/App/Components/Header/Header.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import "./header.css";
+import { Link } from "react-router-dom";
+
+
+class Button extends React.Component {
```

Comment:
Usually it's a good idea to have a single component per file.

In this particular case, I would not put button in it's own component. It's used only once in this app and it's very simple, so I would just keep `<Link...</Link>` part in `Header` component